Formula/apache-ctakes.rb

```diff
@@ -1,8 +1,9 @@
 class ApacheCtakes < Formula
   desc "NLP system for extraction of information from EMR clinical text"
   homepage "https://ctakes.apache.org"
-  url "https://apache.claz.org/ctakes/ctakes-4.0.0.1/apache-ctakes-4.0.0.1-bin.tar.gz"
-  sha256 "f741016e3755054876f3bb27f916a8008af27175ef33785638a6292d300c972e"
+  url "https://dlcdn.apache.org//ctakes/ctakes-4.0.0.1/apache-ctakes-4.0.0.1-src.tar.gz"
+  sha256 "a4e0eb3094bc8dc134487548d4f2b49aa7b8cda41587e4300f34d64179f78271"
+  revision 1
```

Comment:
We can also add license in here
```suggestion
  license "Apache-2.0"
  revision 1
```

license ref, https://svn.apache.org/repos/asf/ctakes/trunk/LICENSE